app/helpers/authentication_helper.rb

```diff
@@ -46,7 +46,7 @@ def waiting_on_first_user?
   end
 
   def private_forem_or_no_enabled_auth_options
```

Comment:
I feel as though this method name should be changed _back_ to `invite_only_mode_or_no_enabled_auth_options` now that we call `ForemInstance.invitation_only?` within it and could be done in a follow-up PR since there are a handful of files that will also need to be changed (roughly 4, not including related specs).

Reply:
I agree @juliannatetreault. We really should try to avoid all confusion between the concepts of "private" and "invitation-only".

Reply:
Update: I decided to add the method name change to this PR to avoid rebasing and since I'm already working in these files. 😊